openslides_backend/presenter/check_database.py

```diff
@@ -39,40 +78,35 @@ def get_result(self) -> Any:
             msg += f" Missing permission: {OrganizationManagementLevel.SUPERADMIN}"
             raise PermissionDenied(msg)
 
-        meeting_ids = self.get_all_meeting_ids()
-        errors: Dict[int, str] = {}
-        for meeting_id in meeting_ids:
-            export = export_meeting(self.datastore, meeting_id)
-            checker = Checker(
-                data=export,
-                mode="internal",
-                repair=True,
-                fields_to_remove={
-                    "motion": [
-                        "origin_id",
-                        "derived_motion_ids",
-                        "all_origin_id",
-                        "all_derived_motion_ids",
-                    ]
-                },
-            )
-            try:
-                checker.run_check()
-            except CheckException as ce:
-                errors[meeting_id] = str(ce)
+        export = self.get_all_database()
+        checker = Checker(
+            data=export,
+            mode="all",
+        )
+        errors: str = ""
+        try:
+            checker.run_check()
+        except CheckException as ce:
+            errors = str(ce)
         if not errors:
             return {"ok": True, "errors": ""}
-        return {"ok": False, "errors": self.gen_error_message(errors)}
+        return {"ok": False, "errors": errors}
```

Comment:
you can combine this:
```
try:
    checker.run_check()
    return {"ok": True}
except CheckException as ce:
    return {"ok": False, "errors": str(ce)}
```

Reply:
Okay, done. I have also updated the presenter wiki.